### ISSUE TITLE
Demo broken

### ISSUE THREAD

**Issue by abulka** [HubSpot/vex#237]
Tried the demo today in both Chrome and Safari and all I got was a black screen after I hit the "demo" button.
http://github.hubspot.com/vex/docs/welcome/

**Comment by bbatliner** [comment]
Huh, me too. That's not good! Any ideas what's going wrong?

**Comment by Dracky3k** [comment]
I would wager a guess that the overlay background is popping forward on the z-index too much and also its stacking cause of the multiple modal windows

**Comment by COLABORATI** [comment]
same here on chromium ubuntu16.04

**Comment by phra** [comment]
![screenshot from 2017-09-07 11-07-47](https://user-images.githubusercontent.com/984628/30155387-f4f09668-93bc-11e7-8c75-7eb432732a40.png)

**Comment by bbatliner** [comment]
I made a workaround in ff15cf5. Basically the overlay is now added directly under the body, so using a different `appendLocation` other than the body will cause the dialogs to appear behind the overlay, unless that appendLocation also has a z-index greater than the overlay.

@timmfin Not sure if you rebuilt the docs from #241 already, but since this changes the demo index.html we'll need to rebuild again, right?
